pkg/skoop/network/aliyun/assertion.go

```diff
@@ -141,14 +144,47 @@ func (a *vpcAssertion) AssertSecurityGroup(srcECS, dstECS string, pkt *model.Pac
 		}
 
 		if !checkResult {
-			sgIDs := lo.MapToSlice(srcECSSecurityGroup,
-				func(id string, _ aliyun.SecurityGroupRule) string { return id })
+			sgIDs := lo.Keys(srcECSSecurityGroup)
 			suspicions = append(suspicions, model.Suspicion{
 				Level: model.SuspicionLevelFatal,
 				Message: fmt.Sprintf("%s(%s) security group %v not allow packet from to %s:%d",
 					srcECS, a.getECSIP(srcECSInfo), sgIDs, pkt.Dst.String(), pkt.Dport),
 			})
 		}
+	} else if srcECS == "" && dstECS != "" {
+		dstECSInfo, err := a.cloudManager.GetECSInfo(dstECS)
+		if err != nil {
+			return nil, err
+		}
+
+		dstENI := a.getENIFromIP(dstECSInfo, pkt.Dst.String())
+		var dstECSSecurityGroup map[string]aliyun.SecurityGroupRule
+		if dstENI == nil {
+			dstECSSecurityGroup = dstECSInfo.Network.SecurityGroups
+		} else {
+			dstECSSecurityGroup = dstENI.SecurityGroups
+		}
+
+		sgs := lo.Values(dstECSSecurityGroup)
+		if len(sgs) > 0 {
+			sgPolicyVerdict := securityPolicyVerdictDrop
+			if sgs[0].Type == aliyun.SecurityGroupTypeNormal {
+				sgPolicyVerdict = securityPolicyVerdictAccept
```

Comment:
for ingress direction, default policy verdict always be drop ever type is normal or enterprise.

Reply:
Fixed.